CommonLibSF/src/RE/T/TESObjectREFR.cpp

```diff
@@ -9,6 +9,13 @@ namespace RE
 		return func(this);
 	}
 
+    std::int32_t TESObjectREFR::GetValue()
```

Comment:
indent issue?

Reply:
clang format action should take care of that when pushed.